gnome-extra/gnome-commander/gnome-commander-1.14.0.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
What about EAPI-8?

Reply:
I will check. Why does `repoman -dx full` does not complain?

Reply:
It will start complaining about it once EAPI-7 becomes deprecated - we are quite some time away from that, but no reason to wait for porting either. ;)

Reply:
Aha :+1: